volume-providers/vpc/vpcclient/models/volume.go

```diff
@@ -48,17 +48,24 @@ type Volume struct {
 
 // ListVolumeFilters ...
 type ListVolumeFilters struct {
-	ResourceGroupID string
-	Tag             string
-	ZoneName        string
-	VolumeName      string
+	ResourceGroupID string `json:"resource_group.id,omitempty"`
+	Tag             string `json:"tag,omitempty"`
+	ZoneName        string `json:"zone.name,omitempty"`
+	VolumeName      string `json:"name,omitempty"`
 }
 
 // VolumeList ...
 type VolumeList struct {
-	Volumes    []*Volume `json:"volumes,omitempty"`
-	Limit      int       `json:"limit,omitempty"`
-	TotalCount int       `json:"total_count,omitempty"`
+	First      *HReference `json:"first,omitempty"`
+	Next       *HReference `json:"next,omitempty"`
+	Volumes    []*Volume   `json:"volumes"`
+	Limit      int         `json:"limit,omitempty"`
+	TotalCount int         `json:"total_count,omitempty"`
+}
+
+// HReference ...
+type HReference struct {
```

Comment:
is there any other variable expected in this struct? if not then we can use same, also `Href` is mandatory in RIaaS struct

Reply:
I did not get wht do u mean by `use same`.
`First` and `Next` fields are of below form `"first":{"href":"https://eu-gb.iaas.cloud.ibm.com/v1/volumes?start=c4fb4ee2-0414-42ab-b438-d2daede61fbb\u0026limit=50"}`
so I think we have to use above way only.